Makefile

```diff
@@ -6,6 +6,7 @@ aro: generate
 
 az:
 	cd python/az/aro && python ./setup.py bdist_egg
+	cd python/az/aro && python ./setup.py bdist_wheel || true
```

Comment:
`|| true` will be removed later when fewer people are installing the extension via cloning this repo.